### DISCUSSION TITLE
Sequence Errors

### DISCUSSION THREAD

**Discussion by przemeksw** [phaag/nfdump#492] [General]
Hi,
What can cause Sequence Errors: 2 messages and how can we prevent it?
I send data from two routers to the collector and collect it with one collector on one port.
One router is linux using iptables and the other is mikrotik.
I also don't understand what exactly this sequence error means for me - loss of some PIS?

**Comment by phaag** [comment]
Netflow v9 and IPFIX have sequence counters to verify the correct sequence of packets. If you have sequence errors, it means that you are missing packets and therefore flows. There may be various reasons for this along the way from the exporter to the collector.

**Reply by przemeksw** [reply]
the network is 100% ok
I also turned off flow control just in case.
Data is exported from Mikrotik and Linux

Maybe some settings on Mikrotik?
Is it possible to determine where these holes occur and from which exporter?

**Reply by phaag** [reply]
Check for `nfdump -E  <file>` to see the exporter statistics and sequence failures. 

You need to check each element of the chain, which may potentially drop packets
